src/app/zap-templates/chip-templates.json

```diff
@@ -41,6 +41,26 @@
             "path": "templates/chip/encoder-src.zapt",
             "name": "CHIP ZCL API",
             "output": "src/app/encoder.cpp"
+        },
+        {
+            "path": "templates/chip/im-encoder-src.zapt",
+            "name": "CHIP Interaction Model Encoder API",
+            "output": "src/app/im-encoder.cpp"
+        },
+        {
+            "path": "templates/chip/im-encoder-h.zapt",
+            "name": "CHIP Interaction Model Encoder API",
+            "output": "src/app/im-encoder.h"
+        },
+        {
+            "path": "templates/chip/python-ChipDeviceController-ClusterCommands.zapt",
+            "name": "CHIP ZCL API for CPython",
+            "output": "src/controller/python/ChipDeviceController-ClusterCommands.cpp"
```

Comment:
Already discussed this with @vivien-apple, but Application specific files shouldn't be stored directly in `/src/`. If those files are not application specific they can be store in '/src/` but within a `/gen/` folder. Point here is to make it obvious that some of those file are generated. That way end user and other developers won't be tempted to modify them manually since they can be regenerated at any time.